### PR TITLE
Feature/laravel mix v4

### PR DESCRIPTION
**Pull request by dsktschy**
This changes make this plugin compatible with Laravel Mix v4.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "laravel-mix-pug",
-  "version": "0.2.0",
+  "version": "0.2.1",
```
Comment (dsktschy) [inline package.json:3]: This line is same state as master branch.
The target branch of this pull request is develop.
Please merge master branch to develop branch, before accepting this pull request.

**Comment by amiri27** [comment]
Any chance to merge this soon?

**Comment by amiri27** [comment]
Any update regarding this?

**Comment by matejsvajger** [comment]
Sorry guys, was away for a while. Looks good.

Thanks @dsktschy for fixing this.

**Comment by dsktschy** [comment]
Thanks @matejsvajger for marging !

**Comment by amiri27** [comment]
Thanks @matejsvajger

**Pull request merged**